scripts/import_with_env.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Complete import script with environment variables
+# Usage: ./scripts/import_with_env.sh [--fresh|--reset] [--pages N]
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)